src/NServiceBus.Core.Tests/Timeout/InMemoryTimeoutPersisterThreadTests.cs

```diff
@@ -26,7 +26,7 @@ public void Run()
 
             for (var i = 0; i < 10; i++)
             {
-                var thread = new Thread(() => Runner(inMemoryTimeoutPersister, options));
+                var thread = new Thread(() => Runner(inMemoryTimeoutPersister, options).Wait());
```

Comment:
Yes we could, as I tried that. But the test is called ThreadTest so I'm not sure what the reason behind the test really is, so I left it as is.